src/filter.rs

```diff
@@ -76,3 +76,181 @@ impl Filter {
         }
     }
 }
+
+#[cfg(test)]
+mod test {
+
+    // TODO Brian: Find a unique access point for the following fn's, repeated in messages.rs.
+    fn generate_random_u8() -> u8 {
```

Comment:
rand::Rng is used within any test that requires it, see the next line.

@benjaminbollen said, try shorter L51 types. ?